lib/passport-mock/factories/legalshield.js

```diff
@@ -0,0 +1,101 @@
+var utils = require('./utils');
+var faker = require('faker');
+
+module.exports = function Legalshield (params) {
+  params = params || { };
+  params.address = params.address || {};
+  params.fulfillment_partner = params.fulfillment_partner || {};
+  params.emergency_contact = params.emergency_contact || {};
+  params.access_control = params.access_control || {};
+
+  var id = (params.id || utils.makeNumber(10)) ;
+  var firstName = (params.firstName || faker.name.firstName());
+  var lastName = (params.lastName || faker.name.lastName());
+  var displayName = (params.displayName || firstName + ' ' + lastName);
+  var email = (params.email || utils.makeEmail(firstName, lastName));
+  var json = {
+    Member: {
```

Comment:
this key isn't a part of the API response, it's body is what's in the API response